### DISCUSSION TITLE
cardano-node version numbering

### DISCUSSION THREAD

**Discussion by shawnim** [IntersectMBO/cardano-node#5044] [General]
Not that important but I find the cardano-node version numbering to be somewhat confusing.
I would expect that the first number would change when their is a hard fork / protocol version upgrade, the second number would change when there is an added feature and the third number would change for bug fix releases.
Is there a method to the current version numbering that I am missing?

**Comment by disassembler** [comment]
That's exactly what we're doing with the new versioning. The next major version will be 8.0.0 (until it's deemed suitable for mainnet we'll append -pre). Until Conway is the stable protocol version well have 8.1.0, 8.2.0, 8.3.0, etc... Not every release will drop the pre suffix. This allows us to continuously roll releases out even if there are regressions in performance for instance, and retag releases when they're deemed stable for mainnet.
